### ISSUE TITLE
NeXus manual not found online now

### ISSUE THREAD

**Issue by prjemian** [nexusformat/definitions#1129]
https://manual.nexusformat.org/user_manual.html results in 404 error (not found)

https://manual.nexusformat.org/ results in 404 error (not found)

https://nexusformat.github.io/definitions displays as expected.

**Comment by FreddieAkeroyd** [comment]
Any idea when this started happening? `manual.nexusformat.org` is just a dns alias for `nexusformat.github.io` so it feel like something may have changed at the github pages end? It seems that  `https://nexusformat.github.io` is no longer serving the definitions root, maybe there is a new setting that needs applying?

**Comment by prjemian** [comment]
Started with #1128.  That might be the place to start looking.  Thanks, @FreddieAkeroyd !

**Comment by FreddieAkeroyd** [comment]
maybe we have lost the custom domain from https://github.com/nexusformat/definitions/settings/pages ? i can look tomorrow if you haven't solved it, I need to get some sleep now :-D

**Comment by prjemian** [comment]
Hmmm...  It's blank now.  Wonder if just entering "https://manual.nexusformat.org/" will solve?  I'll try now.

**Comment by prjemian** [comment]
We need a CNAME file in the repository root according to [these helpful docs](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages).

**Comment by prjemian** [comment]
```bash
(base) prjemian@zap:~$ nslookup  manual.nexusformat.org
Server:         127.0.0.53
Address:        127.0.0.53#53

Non-authoritative answer:
manual.nexusformat.org  canonical name = nexusformat.github.io.
Name:   nexusformat.github.io
Address: 185.199.108.153
Name:   nexusformat.github.io
Address: 185.199.111.153
Name:   nexusformat.github.io
Address: 185.199.110.153
Name:   nexusformat.github.io
Address: 185.199.109.153
Name:   nexusformat.github.io
Address: 2606:50c0:8000::153
Name:   nexusformat.github.io
Address: 2606:50c0:8001::153
Name:   nexusformat.github.io
Address: 2606:50c0:8002::153
Name:   nexusformat.github.io
Address: 2606:50c0:8003::153
```

**Comment by prjemian** [comment]
Our manual is published to: [nexusformat.github.io/definitions](https://nexusformat.github.io/definitions) so slightly different.

**Comment by prjemian** [comment]
We are publishing with this code: https://github.com/nexusformat/definitions/blob/d9574a8f90626a929c677f1505729d1751170989/.github/workflows/publish-sphinx.yml#L79-L83

Looks [possible to redirect](https://github.com/sphinx-notes/pages#inputs) using `target_path`.

**Comment by prjemian** [comment]
Syntax is not completely obvious, will try with `..`.

Note that:

```py
(bluesky_2022_3) prjemian@zap:~/Documents/sandbox$ nslookup nexusformat.org
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	nexusformat.org
Address: 130.246.39.142
```

This is the ISIS Computing Server for web VMs.

**Comment by sanbrock** [comment]
added CNAME as described in the link from @prjemian, and as @FreddieAkeroyd did in Jan 23, 2020

**Comment by prjemian** [comment]
I agree, the https://manual.nexusformat.org/ website is found online again.  @sanbrock Thanks!
